README.md

```diff
@@ -1,91 +1,120 @@
+<div align="center">
+
+  <br/>
+
+  <h3><b>React-Redux Capstone Project</b></h3>
+
+</div>
 
 # 📗 Table of Contents
 
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-- [💻 Getting Started](#getting-started)
-  - [Prerequisites](#prerequisites)
-  - [Setup](#setup)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [📝 License](#license)
-
-# 📖 REACT-REDUX-CAPSTONE<a name="about-project"></a>
-
-This webpage shows a sneak peek into the project I have developed and deployed, showcasing my skills in HTML CSS, Javascript and webpack.
-
-**[React-Redux-Capstone]** is an HTML, CSS and Javascript project
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 capstone ](#-capstone-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [ Video Demo ](#video-demo)
+    - [ Live Link ](#live-link)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖React-Redux Capstone Project<a name="about-project"></a>
+
+This is the capstone project for react. It consumes an API, uses Redux as a state management tool and React for the UI frontend.
 
 ## 🛠 Built With <a name="built-with"></a>
+- React
+- Redux
+- API
 
 ### Tech Stack <a name="tech-stack"></a>
 
-- React.js
-- Redux
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev/">React JS</a></li>
+  </ul>
+</details>
 
 ### Key Features <a name="key-features"></a>
-- **Basic maths Calculations**
-- **Well structured UI**
 
-## 💻 Getting Started <a name="getting-started"></a>
+- **Code Quality**
+- **SPAs**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-To get a local copy up and running, follow these steps.
+### Video Demo <a name="video-demo"></a>
+
+[Video Link](https://www.loom.com/share/d1bb499c11b240ea99069a1166b2e901?sid=66b92fdb-1145-423e-899a-199ed82382b3)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Live Link <a name="live-link"></a>
+
+[Live Link](https://ngalamac.github.io/react-redux-capstone/)
```

Comment:
- [ ] Great job deploying your project with GitHub Pages! 😄 However, I noticed that when I visited the link, it displayed the README instead of the expected content. It seems like there might be a small bug that needs fixing. Don't worry; I'm here to help you resolve it! 🛠️

One alternative solution you might consider is using [Render](https://render.com/docs/deploy-create-react-app) for deployment. It's a more straightforward platform for deploying React apps, and it might save you some time and effort. 

Let me provide a bit more detailed guidance on how you can fix the issue with your current GitHub Pages deployment:

1. **Check your Repository Settings:**
   First, make sure your repository settings are configured correctly. Ensure that the branch you want to deploy (usually 'main' or 'master') is selected under the "GitHub Pages" section in your repository settings.

2. **Verify Your Build Script:**
   Double-check your project's build script in the `package.json` file. The `"homepage"` field should be set to your GitHub Pages URL, like this:
   ```json
   "homepage": "https://yourusername.github.io/your-repo-name",
   ```

3. **Build and Deploy Again:**
   Run the build command for your React app, typically `npm run build`, to generate the production-ready build files. Then, commit and push these changes to your repository. GitHub Pages will automatically deploy the content from the `build` folder.

4. **Ensure Your Default Page:**
   Ensure that your React app's main page is named `index.html` and is located in the root of your `build` folder. GitHub Pages will serve this as the default page.

5. **Verify Repository Structure:**
   Confirm that your project's folder structure is correct. It should look like this:
   ```
   your-repo-name/
   ├── build/
   │   ├── index.html
   │   ├── static/
   │   │   ├── ...
   │   └── ...
   ├── src/
   ├── ...
   ├── package.json
   ├── README.md
   └── ...
   ```

6. **Clear Browser Cache:**
   Sometimes, the old version of your website might be cached in your browser. Try accessing the URL in a private or incognito window or clear your browser cache.

7. **Wait for GitHub Pages to Update:**
   It might take a few minutes for GitHub Pages to update and reflect your changes. Be patient, and check your link again.

If you follow these steps carefully, you should be able to fix the issue with your current GitHub Pages deployment. However, if you prefer a more straightforward approach, you can explore using [Render](https://render.com/docs/deploy-create-react-app) as an alternative. It's known for its simplicity in deploying React apps. Good luck with your project! 🚀

> Your current live project screenshot
![Screenshot from 2023-09-14 11-48-43](https://github.com/ngalamac/react-redux-capstone/assets/112261237/28e7aea6-e4f9-4e9c-9382-a5c6ee0019b8)